README.md

```diff
@@ -1,24 +1,166 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <img src="https://img.icons8.com/color/48/null/coins.png" alt="logo" width="110"  height="auto" />
+  <h1><b>TrackerCoin</b></h1>
 
-* Ruby version
+</div>
 
-* System dependencies
 
-* Configuration
+<!-- TABLE OF CONTENTS -->
 
-* Database creation
+# 📗 Table of Contents
 
-* Database initialization
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  <!-- - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* How to run the test suite
+<!-- PROJECT DESCRIPTION -->
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📖 TrackerCoin <a name="about-project"></a>
 
-* Deployment instructions
+**TrackerCoin** is a Ruby on Rails application that allows you to easily track your transactions and keep them organized by category. The app stores all data separately for each user, ensuring privacy and security.
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+- `Ruby on Rails`
+- `psql`
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- User authentication: Users can sign up, sign in, and sign out to access their own unique data.
+- Transaction tracking: Users can log transactions with names, amounts, and categories to track their spending.
+- Category management: Users can create, view, edit, and delete categories for organizing their transactions.
+- Transaction history: Users can view a history of all their transactions, filtered by category and/or date range.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://trackercoin.onrender.com)
+- [Video Demo Link](https://drive.google.com/file/d/1e5_s2kyaM1QWkKI5SBnCmcW9ypKatr71/view?usp=sharing)
```

Comment:
- [ ] in your presentation kindly show us a piece of code that interests you or that you are proud of :+1: